doc/integrator/extend_data_model.rst

```diff
@@ -257,33 +257,36 @@ And now the file ``geoportal/<package>_geoportal/admin/views/title.py``:
         def duplicate(self):
             return super().duplicate()
 
+Change the User page in the admin, add it in your configuration ``geoportal/vars.yaml``:
+
+.. code:: yaml
+
+   vars:
+       ...
+       admin_interface:
+           ...
+           exclude_pages:
+             - users
+             - roles
+             - functionalities
+             - interfaces
+           include_pages:
+             - url_path: userdetails
+               model: cartoriviera_geoportal.models.UserDetail
```

Comment:
I don't think we want to mention cartoviera_geoportal here?

Reply:
+1 Thanks :-)